src/v1/sources/shopify/config.js

```diff
@@ -31,10 +31,17 @@ const PIXEL_EVENT_MAPPING = {
   checkout_address_info_submitted: 'Checkout Address Info Submitted',
   checkout_contact_info_submitted: 'Checkout Contact Info Submitted',
   checkout_shipping_info_submitted: 'Checkout Shipping Info Submitted',
-  payment_info_submitted: 'Payment Info Submitted',
+  payment_info_submitted: 'Payment Info Entered',
   search_submitted: 'Search Submitted',
 };
 
+const RUDDER_ECOM_MAP = {
+  checkouts_create: 'Checkout Started Server',
```

Comment:
`Checkout Started Webhook` makes more sense ?

Reply:
Yes, LMK if we should do it

Reply:
As discussed. updated to Checkout Started - Webhook